app/README.md

```diff
@@ -0,0 +1,19 @@
+
+
+# application code
+
+## starting the app
+
+This section contains details on starting up the application. For now, we are using `docker-compose` for orchestration, just to speed up development.
+
+Kick up the app:
+
+```
+docker-compose up -d
```

Comment:
this _almost_ works. Some filepath stuff inside the flask container is getting messed up. Let me know if you guys see anything weird in the docker-compose.yml. Otherwise I'll figure it out tomorrow

Reply:
FYI this is working now

Reply:
interesting - dont you have to do a ```docker-compose up build``` so that the flask app gets build ?
```
+services:
 +  flaskapp:
 +    build: 
 +      context: ./ui
 +      dockerfile: Dockerfile-ui
 +    ports:
 +      - "5090:5090"
 +    networks:
 +      - web_nw
```

Reply:
I tried pulling this branch a week back and did not work - will try again now

Reply:
you need to do `docker-compose build` if you've built the containers previously and anything has changed (since `docker-compose up` won't rebuild containers)

fair point, I should clarify